deployment/CP03ISSM_Deploy.csv

```diff
@@ -279,65 +279,65 @@ AR-31,,AR-34,,CP03ISSM-SBD11-06-METBKA000,9,1,2018-10-29T03:43:00,2019-04-06T18:
+AR-39,,,,CP03ISSM-MFD37-03-CTDBPD000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-CTDBPD-50008,40.3594,-70.8851,,92,93,
+AR-39,,,,CP03ISSM-MFD37-04-DOSTAD000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-DOSTAD-00386,40.3594,-70.8851,,92,93,
+AR-39,,,,CP03ISSM-MFD37-07-ZPLSCC000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-ZPLSCC-55100,40.3594,-70.8851,,92,93,
+AR-39,,,,CP03ISSM-RIC21-00-CPMENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-NCPM02-20002,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID26-00-DCLENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-NDCL26-20003,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID26-04-VELPTA000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-VELPTA-12234,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID26-06-PHSEND000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-PHSEND-P0068,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID26-07-NUTNRB000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-NUTNRB-01106,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID26-08-SPKIRB000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-SPKIRB-00269,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID27-00-DCLENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-NDCL27-20002,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID27-01-OPTAAD000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-OPTAAD-00153,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID27-02-FLORTD000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-FLORTD-01318,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID27-03-CTDBPC000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-CTDBPC-50056,40.3594,-70.8851,,7,93,
+AR-39,,,,CP03ISSM-RID27-04-DOSTAD000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-DOSTAD-00385,40.3594,-70.8851,,7,93,
+#AR-39,,,,CP03ISSM-SBC11-00-CPMENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-BCPM01-50004,40.3594,-70.8851,,0,93,s/n same as for currently deployed ISSM
+#AR-39,,,,CP03ISSM-SBD11-00-DCLENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-BDCL11-50004,40.3594,-70.8851,,0,93,s/n same as for currently deployed ISSM
+AR-39,,,,CP03ISSM-SBD11-01-MOPAK0000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-MOPAK0-43257,40.3594,-70.8851,,0,93,
+AR-39,,,,CP03ISSM-SBD11-02-HYDGN0000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-HYDGN0-42316,40.3594,-70.8851,,0,93,
+AR-39,,,,CP03ISSM-SBD11-06-METBKA000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-METLGR-00012,40.3594,-70.8851,,-3,93,
+#AR-39,,,,CP03ISSM-SBD12-00-DCLENG000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGCON-BDCL12-50004,40.3594,-70.8851,,0,93,s/n same as for currently deployed ISSM
+AR-39,,,,CP03ISSM-SBD12-03-HYDGN0000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-HYDGN0-41917,40.3594,-70.8851,,0,93,
+AR-39,,,,CP03ISSM-SBD12-04-PCO2AA000,11,1,2019-09-26T13:50:00,,CGMCP-03ISSM-00011,,CGINS-PCO2AA-31124,40.3594,-70.8851,,0,93,
```

Comment:
Looks good, verified with QLR. We can uncomment lines 336, 336, and 341 if we wanted since we now have recovery times in for the previous deployment. Though we still need to verify the SNs with e-lab.

Reply:
Look ok.  I also viz'd the QLR.

Reply:
@cdobs I updated the platcon on ISSM-11 to use the UNKNOWN UIDs and uncommented lines 336, 337, & 341. Please do final review of those.

Reply:
Looks good!

Reply:
@spetillo , @cdobs – The Platcon updates look good to me.